src/Tools/Console/Command/RunSqlCommand.php

```diff
@@ -75,23 +76,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         assert(is_string($sql));
 
-        $depth = $input->getOption('depth');
-
-        if (! is_numeric($depth)) {
-            throw new LogicException("Option 'depth' must contains an integer value");
+        if ($input->getOption('depth') !== null) {
+            $io->warning('Parameter "depth" is deprecated and has no effect anymore.');
```

Comment:
Should `stderr` be used for all warnings and errors? What happens if we run this command with the json descriptor?

Reply:
I expect this tool to be run by a human being for debug purposes rather that being part of some automation.